docs/api/item.md

```diff
@@ -58,8 +59,19 @@ See the [theming documentation](/docs/theming/css-variables) for more informatio
 -->
 
 
+## Item Lines
+
+Items show an inset bottom border by default. The border has padding on the left and does not appear under any content that is slotted in the `"start"` slot. The `lines` property can be modified to `"full"` or `"none"` which will show a full width border or no border, respectively.
+
+import Lines from '@site/static/usage/item/lines/index.md';
+
+<Lines />
+
+
 ## Item Placement
 
+### Slotted Content
+
 Item uses named [slots](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot) in order to position content. This logic makes it possible to write a complex item with simple, understandable markup without having to worry about styling and positioning the elements.
 
 The below chart details the item slots and where it will place the element inside of the item:
```

Comment:
I'm good with removing the section on slots, agree that it is redundant. Should we also remove the `Text Alignment` section under it? I believe we originally added this section because people were having trouble with wrapping the text in an item, so it was a way to link them to the docs on `.ion-text-wrap`, but I did use this class in the basic usage example so maybe that is sufficient now? The only other components where we link to the CSS Utilities are the content & grid components, and I can't see any other classes in there that are specifically helpful to item.

Reply:
Good call, I agree with removing that section as well 👍 If we want to be safe, we could always add a callout for the text wrapping class above the Basic playground. That should make it easier to find while still cleaning things up.